qa/suites/rgw/multifs/omap_limit/10.yaml

```diff
@@ -0,0 +1,5 @@
+overrides:
+  ceph:
+    conf:
+      osd:
+        osd_max_omap_entries_per_request: 10
```

Comment:
this setting causes swift tests to fail with `FAIL: testUtf8Container (test.functional.tests.TestContainer)` (see [a teuthology.log](http://qa-proxy.ceph.com/teuthology/yuriw-2017-07-11_21:12:18-rgw-wip-yuri-testing2_2017_7_12-distro-basic-smithi/1388789/teuthology.log)). it appears to list only the first 10 buckets